### PR TITLE
Suppress error output on master check

### PR DESCRIPTION
**Pull request by ikitiki**

### CODE REVIEW

File: patroni/ha.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ def fetch_node_status(member):
             xlog_location = None if is_master else json['xlog']['replayed_location']
             return (member, True, not is_master, xlog_location, json.get('tags', {}))
         except:
-            logger.exception('request failed: GET %s', member.api_url)
+            logger.error("request failed: GET %s", member.api_url)
```
Comment (CyberDem0n) [inline patroni/ha.py]: exception was also producing ERROR message, but with the stacktrace.
Annoying part was not stacktrace itself, but ERROR. It should be replaced with warning and probably it also make sense to show exact reason why it has failed:
```python
except Exception as e:
    logger.error("GET %s request failed: %s", member.api_url, e)
```

**Comment by CyberDem0n** [comment]
:+1:

**Comment by ikitiki** [comment]
👍

**Comment by ikitiki** [comment]
:+1:

**Comment by ikitiki** [comment]
:+1:

**Pull request merged**